platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -2,9 +2,277 @@
     'use strict';
 
     var uiServices = angular.module('uiServices', ['ngResource']);
-
     uiServices.factory('Menu', function($resource) {
         return $resource('module/menu');
     });
 
+    uiServices.service('BootstrapDialogManager', function () {
+        var modals2 = [];
```

Comment:
Can we come up with a more descriptive variable name then 'modals2'?

Suggestions might include:
'modals'
'modalsList'
'modalWindows'
'modalWindowStack'